pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -43,12 +43,19 @@ const (
 
 	// OperatorNotInitializedMessage is the message we print when the Operator is not ready to reconcile, typically the ceph.conf has not been generated yet
 	OperatorNotInitializedMessage = "skipping reconcile since operator is still initializing"
+
+	// CancellingOrchestrationMessage is the message to indicate a reconcile was cancelled
+	CancellingOrchestrationMessage = "CANCELLING CURRENT ORCHESTRATION"
 )
 
 var (
 	// ImmediateRetryResult Return this for a immediate retry of the reconciliation loop with the same request object.
 	ImmediateRetryResult = reconcile.Result{Requeue: true}
 
+	// ImmediateRetryResultNoBackoff Return this for a immediate retry of the reconciliation loop with the same request object,
+	// but don't wait for exponential backoff
+	ImmediateRetryResultNoBackoff = reconcile.Result{Requeue: true, RequeueAfter: time.Second}
```

Comment:
@travisn   can you elaborate on how this prevents waiting for exponential backoff. Any references would be useful. I maybe wrong, but I thought `Requeue: true` would always retry with exponential backoff.

Reply:
By setting the `RequeueAfter` field, the exponential backoff behavior is overridden. I don't have a doc reference, but was following the example of the setting `WaitForRequeueIfCephClusterNotReady` seen after this.